instrumentation/aws_sdk/lib/opentelemetry/instrumentation/aws_sdk/handler.rb

```diff
@@ -0,0 +1,73 @@
+# frozen_string_literal: true
+
+# Copyright The OpenTelemetry Authors
+#
+# SPDX-License-Identifier: Apache-2.0
+
+module OpenTelemetry
+  module Instrumentation
+    module AwsSdk
+      # Generates Spans for all interactions with AwsSdk
+      class Handler < Seahorse::Client::Handler
+        def call(context)
+          span_name = get_span_name(context)
+          attributes = get_span_attributes(context)
```

Comment:
The specification requires that we set `db.system: dynamodb` in the event that this is a request for dynamodb. I think this would be possible to determine based on the context 

https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/trace/semantic_conventions/instrumentation/aws-sdk.md#dynamodb

Could we add support+a test for this as it's required (if there are optional attributes you think are worth adding from dynamoDB, the more the merrier, but this is the only one that's required as per the specification)?

https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/trace/semantic_conventions/instrumentation/aws-sdk.md#dynamodb

Reply:
@ericmustin Nice catch! 
[Added `db.system` attribute](
https://github.com/open-telemetry/opentelemetry-ruby/pull/1014/commits/cfd81abada4f85dbdbaaa893729b668ec8f7e198)